README.md

```diff
@@ -6,12 +6,16 @@ To test it:
 Get Status of running jobs:
 curl http://localhost:5000/jobstatus
 
-To validate a job:
+You'll need a bearer token to validate and queue jobs. 
```

Comment:
@michaelvidal24 Added some instructions per your suggestion, will this work?

Reply:
Looks good, one change you might consider is setting up a specific databus client that has only the scopes needed to interact with the database, that way you stick to the least privilege principle.